src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -69,7 +69,7 @@ def load_arguments(self, _):
         c.argument('code_dir', options_list=['--code-dir'], help='The directory to upload bot code from.')
         c.argument('proj_file_path', options_list=['--proj-file-path', c.deprecate(target='--proj-name',
                                                                                    redirect='--proj-file-path',
-                                                                                   hide=True, expiration='3.0.0')],
```

Comment:
@stevengum, please kindly take a look at the changes and take action accordingly. Thanks.